src/users/edit.njk

```diff
@@ -45,11 +45,14 @@
       }) }}
     {% endif %}
 
-    <form method="post" action="{{ linkTo('admin.organizations.users.invite.resend', {organizationGUID: organization.metadata.guid, userGUID: user.metadata.guid}) }}" class="govuk-!-mt-r6">
-      {{ govukButton({
-        text: "Resend user invite"
-      }) }}
-    </form>
+    {% if not isActive %}
+      <form method="post" action="{{ linkTo('admin.organizations.users.invite.resend', {organizationGUID: organization.metadata.guid, userGUID: user.metadata.guid}) }}" class="govuk-!-mt-r6">
+        <p>It'd appear the user did not setup their account yet.</p>
```

Comment:
Think this should at least be "it would."  The content designers have new rules about never using shortened versions of things as they found that it confused some people.

Reply:
Cool. Sounds good. Will change.

Reply:
Personally I do not really get that sentence "It would appear". But I am not a native speaker.

Would it better something like "Apparently the user has not activated the account yet."?